[2024-02-23T03:34:03.994+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Data-pipeline.extract_employee scheduled__2024-02-22T00:00:00+00:00 [queued]>
[2024-02-23T03:34:03.999+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Data-pipeline.extract_employee scheduled__2024-02-22T00:00:00+00:00 [queued]>
[2024-02-23T03:34:03.999+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-23T03:34:04.006+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_employee> on 2024-02-22 00:00:00+00:00
[2024-02-23T03:34:04.011+0000] {standard_task_runner.py:60} INFO - Started process 311 to run task
[2024-02-23T03:34:04.015+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Data-pipeline', 'extract_employee', 'scheduled__2024-02-22T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpmeb6e9h5']
[2024-02-23T03:34:04.018+0000] {standard_task_runner.py:88} INFO - Job 7: Subtask extract_employee
[2024-02-23T03:34:04.184+0000] {task_command.py:423} INFO - Running <TaskInstance: Data-pipeline.extract_employee scheduled__2024-02-22T00:00:00+00:00 [running]> on host 313a02c13f91
[2024-02-23T03:34:05.734+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Data-pipeline' AIRFLOW_CTX_TASK_ID='extract_employee' AIRFLOW_CTX_EXECUTION_DATE='2024-02-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-22T00:00:00+00:00'
[2024-02-23T03:34:05.859+0000] {logging_mixin.py:188} INFO - Error: (psycopg2.errors.UniqueViolation) duplicate key value violates unique constraint "pg_type_typname_nsp_index"
DETAIL:  Key (typname, typnamespace)=(raw_employees, 2200) already exists.

[SQL: 
CREATE TABLE raw_employees (
	employe_id BIGINT, 
	branch_id BIGINT, 
	salary BIGINT, 
	join_date TEXT, 
	resign_date TEXT
)

]
(Background on this error at: https://sqlalche.me/e/14/gkpj)
[2024-02-23T03:34:05.861+0000] {python.py:201} INFO - Done. Returned value was: employees
[2024-02-23T03:34:05.915+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Data-pipeline, task_id=extract_employee, execution_date=20240222T000000, start_date=20240223T033403, end_date=20240223T033405
[2024-02-23T03:34:05.999+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
